src/components/TimelineComponent.tsx

```diff
@@ -810,11 +806,12 @@ export default class TimelineComponent
    * @param {any} selection: the selection for the object to draw
    */
   drawEventMagnitude(selection: any): void {
+    const spacing = 48.5; // this needs to change
```

Comment:
This needs to change

Reply:
Yes, it works for now however is a bit hacky, leaving myself or someone else a note for updating the data with the labels.

Reply:
Then can you clarify that in the code? So when another developer comes across a line that says `This needs to change` we know why?

Reply:
Oh you already did, my bad